src/main/scala/cromwell/engine/CopyWorkflowOutputs.scala

```diff
@@ -0,0 +1,21 @@
+package cromwell.engine
+
+import cromwell.binding._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object CopyWorkflowOutputs {
+  val Name: LocallyQualifiedName = "$final_call$copy_workflow_outputs"
+}
+
+/**
+  * Final call implementation that copies workflow outputs to a specified destination.
+  */
+case class CopyWorkflowOutputs(workflow: WorkflowDescriptor) extends FinalCall {
+  override def unqualifiedName = CopyWorkflowOutputs.Name
+
+  override def rootWorkflow: Workflow = workflow.namespace.workflow
+
+  /** TODO replace snarky println with call to Thibault's code that actually copies files. */
```

Comment:
Oh I don't know, leaving this in (or perhaps the jetski comment) would be fine :+1: